resources/sass/components/_forms.scss

```
@@ -66,3 +66,27 @@
   }
 }
 
+
+.search-dropdown {
+  display: flex;
+  position: static;
+  .dropdown-menu[data-bs-popper] {
+    left: auto;
+    right: calc(var(--bs-gutter-x)*.5);
+    margin-left: calc(var(--bs-gutter-x)*.5);
```

Code style, I think we want space around operators.
I have not found an actual rule, but the example code has spaces around the operator:
https://www.drupal.org/docs/develop/standards/css/css-formatting-guidelines
And it's more readable.
```suggestion
    right: calc(var(--bs-gutter-x) * .5);
    margin-left: calc(var(--bs-gutter-x) * .5);
```